builder/remotecontext/detect.go

```diff
@@ -173,6 +174,9 @@ func StatAt(remote builder.Source, path string) (os.FileInfo, error) {
 func FullPath(remote builder.Source, path string) (string, error) {
 	fullPath, err := remote.Root().ResolveScopedPath(path, true)
 	if err != nil {
+		if runtime.GOOS == "windows" {
+			return "", fmt.Errorf("failed to resolve scoped path %s (%s): %s. Possible cause is a forbidden path outside the build context", path, fullPath, err)
```

Comment:
Perhaps use `errors.Wrapf(err, "failed to .....")`

Reply:
I think that would make it non-sensicle and put the actual error in the wrong place.